packages/react-debug-tools/src/__tests__/ReactHooksInspectionIntegration-test.js

```diff
@@ -618,6 +620,8 @@ describe('ReactHooksInspectionIntegration', () => {
 
     await LazyFoo;
 
+    Scheduler.flushAll();
```

Comment:
It's the synchronous ping thing. I didn't wrap in a feature flag because I can only access feature flags in an `internal` test suite, which means the tests won't run against the production bundles. This particular test is about resolving `defaultProps`, so the difference in how the ping is resolved isn't really relevant.